addons/mail/static/src/models/discuss_sidebar_category/discuss_sidebar_category.js

```diff
@@ -63,6 +63,27 @@ registerModel({
             }
             return clear();
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeAddingItemInputView() {
+            if (
+                this.discussAsChannel &&
+                this.messaging.discuss.view &&
+                this.messaging.discuss.view.sidebarAddChannelInputView
+            ) {
+                return replace(this.messaging.discuss.view.sidebarAddChannelInputView);
+            }
+            if (
+                this.discussAsChat &&
+                this.messaging.discuss.view &&
```

Comment:
Sidebar category is a view (event though it doesn't have the name), so I would be expecting its identifying field to be `discussViewOwner` now, and not have to follow this relation (and even less so access discuss globally).

Reply:
Makes sense to me, will do